scipy/linalg/tests/test_decomp_cholesky.py

```diff
@@ -266,3 +266,116 @@ def test_empty(self, dt):
 
         xx, lower = cho_factor(np.eye(2, dtype=dt))
         assert x.dtype == xx.dtype
+
+
+def gen_input(n=1000, seed=72479):
+    rng = np.random.default_rng(seed)
+    A = rng.random((n, n))
+    A = A + A.T + n * np.eye(n)
+    R = cholesky(A)
+    z = rng.random(n)
+    # Uncomment the following to make it occasionally trip up
+    #z *= 0.95 * np.sqrt(n)
+    return R, z
+
+
+class TestCholeskyUpdate:
+    def test_update(self):
```

Comment:
If update and downdate really just differ by a sign change, let's @parameterize to consolidate.